index.html

```diff
@@ -274,7 +274,7 @@ <h2>Feasibility</h2>
           <h2>Fingerprinting mitigation levels of success</h2>
           <p>There are different levels of success in mitigating browser fingerprinting:</p>
           <dl>
-          <dt>Decreased fingerprinting surface</dt><dd>Removing the source of entropy or accessible attributes that can be used for fingerprinting.</dd>
+          <dt>Decreased fingerprinting surface</dt><dd>Removing the source of entropy or visible attributes that can be used for fingerprinting.</dd>
```

Comment:
Maybe this should be "available" too, as "visible" might imply visibility to the eye (like features that are hidden or visible via CSS).

Reply:
Fixed